packages/grid/src/__tests__/__e2e__/Grid.cy.tsx

```diff
@@ -39,8 +39,8 @@ const clickCell = (row: number, col: number) => {
 
 const checkRowSelected = (row: number, expectedSelected: boolean) => {
   cy.get(`tr[data-row-index="${row}"]`).should(
-    expectedSelected ? "have.class" : "not.have.class",
-    "saltGridTableRow-selected"
+    expectedSelected ? "have.attr" : "not.have.attr",
```

Comment:
This test only tests on the attribute's presence but not the value. This would mean that <tr aria-selected="false" /> would be picked up as selected.

Reply:
Indeed

Reply:
Updated